src/components/H/H1.tsx

```diff
@@ -0,0 +1,13 @@
+import { cn } from '@/lib/utils'
+import { FC, PropsWithChildren } from 'react'
+import { StylableComponentProps } from '../../components/commonProps'
+
+export type H1Pros = StylableComponentProps<HTMLHeadingElement> & PropsWithChildren
+
+export const H1: FC<H1Pros> = ({ children, className }) => (
```

Comment:
I'm not sure about this additional component, we already have the `Typography` component that can handle this.